src/myr/Myr.js

```diff
@@ -41,7 +41,8 @@ class Myr {
                 this.els[it] = this.baseEls[it];
             });
         }
-
+        this.startEvent = "";
```

Comment:
Can we move this to above `if (baseEls)` so that all the cursor elements are together?